### ISSUE TITLE
[Help]: Phase jumps between bursts

### ISSUE THREAD

**Issue by SteffanDavies** [mobigroup/gmtsar#64]
Dear @mobigroup, analysis of Morocco earthquake showing phase jumps between bursts. I suppose this is from bad alignment, do you have a solution to this problem?

![ramps](https://github.com/mobigroup/gmtsar/assets/50405969/a87f93c0-e1a2-4008-b56b-5984791cc26a)
![ramps_rad](https://github.com/mobigroup/gmtsar/assets/50405969/c602ca05-5527-48f2-a867-b9ce9b76f201)

**Comment by AlexeyPechnikov** [comment]
Please share the list of scenes for the case; I'll check it sometime later.

**Comment by SteffanDavies** [comment]
> Please share the list of scenes for the case; I'll check it sometime later.

S1A_IW_SLC__1SDV_20230903T183344_20230903T183412_050167_0609B4_100E
S1A_IW_SLC__1SDV_20230915T183345_20230915T183413_050342_060F9F_85A4

**Comment by AlexeyPechnikov** [comment]
Yes, I can reproduce the issue you've mentioned. It's possible that the Enhanced Spectral Diversity (ESD) feature could help resolve it, though it's not guaranteed (it is not available in PyGMTSAR for now). I recommend trying it out to see if it offers any improvement in your case.
